devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1037,6 +1201,108 @@ void HumanStateProvider::run()
     //                                          pImpl->linkErrorAngularVelocities);
 }
 
+bool HumanStateProvider::impl::applyRpcCommand()
```

Comment:
I would suggest splitting this function into subfunctions, making it easier to understand every if-else branch.

Reply:
done https://github.com/robotology/human-dynamics-estimation/pull/164/commits/b484dfef85051f67e1a553c6784f672f309a5f7f